src/common/packager.ts

```diff
@@ -3,7 +3,9 @@
 
 import {ChildProcess} from "child_process";
 import {CommandExecutor} from "./commandExecutor";
-import {Log, LogLevel} from "./log";
+import {ErrorHelper} from "./error/errorHelper";
+import {Log} from "./log/log";
+import {LogLevel} from "./log/logHelper";
 import {Node} from "./node/node";
 import {OutputChannel} from "vscode";
```

Comment:
I believe that these classes are "vscode-aware", in that they do things with OutputChannel at the type level but should not actually end up require()ing it at runtime. If we can abstract that detail out by passing around an ILogger instead, that would lead to less chance of accidentally referencing OutputChannel in a way that caused it to be required.